src/sentry/api/serializers/models/event.py

```diff
@@ -285,6 +285,11 @@ def serialize(self, obj, attrs, user):
                 'tags': tags_meta,
             },
         }
+        # transaction events have start and end-times that are
+        # timestamp floats.
+        if obj.get_event_type() == 'transaction':
+            d['startTimestamp'] = obj.data.get('start_timestamp')
+            d['timestamp'] = obj.data.get('timestamp')
```

Comment:
I think this ends up being the same as obj.datetime and the field dateCreated that is populated at line 272. Is there a reason to add a new one ?

Reply:
The `dateCreated` field is a string and lacks the ms precision.

Reply:
then, since this is going to be there only for transacitons, I would advise to rename the two fields in such a way that make it clear what they mean like:
startTimestamp
endTimestamp

So people would not have to look into the code to understand what timestamp means in the context of the transaction.